vehicle/mkvi/software/bms/tasks/openwire.c

```diff
@@ -4,19 +4,115 @@
 #include "vehicle/mkvi/software/bms/bms_config.h"
 #include "vehicle/mkvi/software/bms/can_api.h"
 #include "vehicle/mkvi/software/bms/utils/fault.h"
-
+#include <string.h>
 
 #define NO_OPEN_WIRES (65535)
+#define NUM_CV_REG    (4)
+#define NUM_CHANNELS  (18)
+
+/*
+Reads and parses the LTC681x cell voltage registers.
+The function is used to read the parsed Cell voltages codes of the LTC681x.
+This function will send the requested read commands parse the data
+and store the cell voltages in c_codes variable.
+*/
+uint8_t LTC681x_rdcv_trunc(
```

Comment:
same thing here, if it's already working feel free to leave as-is, generally speaking though I'd prefer just defining an array, calling rdcv_reg and saving the data into our own array like we do in the voltage task instead of trying to use truncated versions of the structs in the library - personally I think it opens up to potentially hard-to-find bugs, especially when we start calling functions that expect cell_asic and pass them truncated cell_asics.